docs/proposals/11-apiserver-network-proxy.md

```diff
@@ -27,7 +27,7 @@ reviewers:
 
 ## Problem
 
-Gardener's architecture for Kubernetes clusters relies on having the control-plane (e.g., kube-apiserver, kube-scheduler, kube-controller-manager, etc.) and the data-plane  (e.g., kube-proxy, kubelet, etc.) of the cluster residing in separate places, this provides many benefits but poses some challenges, especially when API-server to system components communication is required. This problem is solved today in Gardener by [making use of OpenVPN](https://github.com/gardener/vpn) to establish a VPN connection from the seed to the shoot. To do so, the following steps are required:
+Gardener's architecture for Kubernetes clusters relies on having the control-plane (e.g., kube-apiserver, kube-scheduler, kube-controller-manager) and the data-plane  (e.g., kube-proxy, kubelet) of the cluster residing in separate places, this provides many benefits but poses some challenges, especially when API-server to system components communication is required. This problem is solved today in Gardener by [making use of OpenVPN](https://github.com/gardener/vpn) to establish a VPN connection from the seed to the shoot. To do so, the following steps are required:
```

Comment:
Did we not decide [here](https://github.com/gardener/gardener/pull/7488#pullrequestreview-1299183314) to not update GEPs?

Reply:
I did a mass update on certain common grammatical and punctuation mistakes found in the documentation as a whole. Apart from that, I won't be updating the GEPs any further.

Reply:
> Did we not decide [here](https://github.com/gardener/gardener/pull/7488#pullrequestreview-1299183314) to not update GEPs?

I did a mass update on certain common grammatical and punctuation mistakes found in the documentation as a whole. Apart from that, I won't be updating the GEPs any further.